opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchScrollRequest.java

```diff
@@ -151,7 +151,40 @@ public void reset() {
    * @return a string representing the scroll request.
    */
   @Override
-  public String toCursor() {
-    return scrollId;
+  public boolean hasAnotherBatch() {
+    return !needClean && scrollId != null && !scrollId.equals("");
+  }
+
+  @Override
+  public void writeTo(StreamOutput out) throws IOException {
```

Comment:
It looks like serializable, but not deserializable

Reply:
It's implementing OpenSearch's serialization API. Deserialization is done in the `StreamInput` constructor.